core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -490,7 +499,9 @@ final class ShuffleBlockFetcherIterator(
     }
 
     currentResult = result.asInstanceOf[SuccessFetchResult]
-    (currentResult.blockId, new BufferReleasingInputStream(input, this))
+    (currentResult.blockId,
```

Comment:
What is the purpose of this new updated return? currentResult.blockId is already returning in tuple.

Reply:
It is still returning the same tuple. I am just passing more arguments to the constructor of BufferReleasingInputStream